<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.4.0 -->
    <title>Kircher Lab</title>
    <meta name="generator" content="Jekyll v3.7.3" />
    <meta property="og:title" content="BIH Computational Genome Biology &amp; UzL/UKSH Regulatory Genomics (Kircher Lab)" />
    <meta property="og:locale" content="en_US" />
    <link rel="canonical" href="https://kircherlab.github.io/" />
    <meta property="og:url" content="https://kircherlab.github.io/" />
    <meta property="og:site_name" content="BIH Computational Genome Biology &amp; UzL/UKSH Regulatory Genomics (Kircher Lab)" />
    <script type="application/ld+json">
      {"url":"https://kircherlab.github.io/","@type":"WebSite","name":"BIH Computational Genome Biology, UzL/UKSH Regulatory Genomics (Kircher Lab)","headline":"Kircher Lab","@context":"http://schema.org"}
    </script>
    <!-- End Jekyll SEO tag -->
    <link rel="stylesheet" href="/assets/main.css"><link type="application/atom+xml" rel="alternate" href="https://kircherlab.github.io/feed.xml" title="Kircher Lab: Computational Genome Biology &amp; Regulatory Genomics" />
  </head>
  <body>
    <header class="site-header" role="banner">
      <div class="wrapper"><a class="site-title" rel="author" href="/">Computational Genome Biology &amp; Regulatory Genomics</a>
        <nav class="site-nav">
          <input type="checkbox" id="nav-trigger" class="nav-trigger" />
          <label for="nav-trigger">
            <span class="menu-icon">
              <svg viewBox="0 0 18 15" width="18px" height="15px">
                <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
                <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
                <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
              </svg>
            </span>
          </label>
          <div class="trigger">
            <a class="page-link" href="/research.html">Research</a>
            <a class="page-link" href="/projects.html">Projects & Resources</a>
            <a class="page-link" href="/people.html">People</a>
            <a class="page-link" href="/publications.html">Publications</a>
            <a class="page-link" href="/events.html">Events</a>
            <a class="page-link" href="/contact.html">Contact</a>
            <a class="page-link" href="/impressum.html">Impressum</a>
          </div>
        </nav>
      </div>
    </header>
<main class="page-content" aria-label="Content">
    <div class="wrapper">
        <article class="post">

            <header class="post-header">
                <h1 class="post-title">CADD-SV Score</h1>
            </header>

            <div class="post-content">
                <p>
                    In recent years, we saw rapid technological advances in the identification of structural variants
                    (SVs) in the human genome, however the interpretation of these variants remains challenging. Several
                    methods were developed that utilize individual mechanistic principles like the deletion of coding
                    sequence or 3D genome architecture disruptions. However, a comprehensive tool that uses the broad
                    spectrum of available annotations for estimating the effect of SVs and prioritize functional
                    variants in health and disease is missing. Here we introduce CADD-SV, a method to retrieve and
                    integrate a wide set of annotations across the range and in the vicinity of SVs for functional
                    scoring. So far, supervised learning approaches were of very limited power for this kind of
                    application, due to a very small number of functionally annotated (e.g. pathogenic/benign) SV sets.
                    We overcome this problem by using a surrogate training-objective, the combined annotation dependent
                    depletion (CADD) of functional variants in evolutionary derived variant sets. Our tool computes
                    summary statistics and uses a trained linear model to differentiate deleterious from neutral
                    structural variants. We use human and chimpanzee derived alleles as proxy-neutral and contrast them
                    with matched simulated variants as proxy-pathogenic, an approach that has proven powerful in the
                    interpretation of SNVs and short InDels (Kircher & Witten et al, 2014). In a proof-of-principle
                    study, we show that CADD-SV scores correlate with known pathogenic variants in individual genomes as
                    well as allelic diversity observed across the population. </p>
                <p>
                    <a href="https://kircherlab.bihealth.org/download/CADD-SV/">Here</a> we provide pre-scored deletions
                    from the initial <a href="https://www.nature.com/articles/s41586-020-2287-8">gnomAD-SV</a> release.
                </p>
            </div>
        </article>

    </div>
</main>    <footer class="site-footer h-card">
      <data class="u-url" href="/"></data>
      <div class="wrapper">
        <!--    <h2 class="footer-heading">Kircher Lab: Computational Genome Biology &amp; Regulatory Genomics</h2>   -->
        <div class="footer-col-wrapper">
          <div class="footer-col footer-col-1">
            <ul class="social-media-list"><li><a href="https://github.com/kircherlab"><svg class="svg-icon"><use xlink:href="/assets/minima-social-icons.svg#github"></use></svg> <span class="username">kircherlab</span></a></li></ul>
          </div>
          <div class="footer-col footer-col-3">
            <p></p>
          </div>
        </div>
      </div>
    </footer>
  </body>
</html>
