<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.4.0 -->
<title>Research | Computational Genome Biology</title>
<meta name="generator" content="Jekyll v3.7.3" />
<meta property="og:title" content="Research" />
<meta property="og:locale" content="en_US" />
<link rel="canonical" href="https://kircherlab.github.io/research.html" />
<meta property="og:url" content="https://kircherlab.github.io/research.html" />
<meta property="og:site_name" content="BIH JRG Computational Genome Biology (Kircher Lab)" />
<script type="application/ld+json">
{"url":"https://kircherlab.github.io/research.html","@type":"WebPage","headline":"Research","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<link rel="stylesheet" href="/assets/main.css"><link type="application/atom+xml" rel="alternate" href="https://kircherlab.github.io/feed.xml" title="BIH JRG Computational Genome Biology (Kircher Lab)" /></head>
<body><header class="site-header" role="banner">

  <div class="wrapper"><a class="site-title" rel="author" href="/">Computational Genome Biology</a><nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
              <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
              <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger"><a class="page-link" href="/research.html">Research</a><a class="page-link" href="/people.html">People</a><a class="page-link" href="/publications.html">Publications</a><a class="page-link" href="/contact.html">Contact</a></div>
      </nav></div>
</header>
<main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post">

  <header class="post-header">
    <h1 class="post-title">Research Focus</h1>
  </header>

  <div class="post-content">
    <p>Our research focuses on computational approaches of identifying functionally relevant genetic changes in disease and adaptation as well as developing more sensitive methods in diagnostics (especially exome, genome and cell-free DNA sequencing). Generally, our research spans the fields of sequence analysis, data mining, machine learning and functional genomics.</p>

<h3 id="research-interest">Research Interest</h3>

<p>Based on a broad interest in genetics, epigenetics, and human adaptation, the group develops computational solutions to overcome technical and experimental obstacles in high-throughput sequencing-based protocols. Our main focus area are computational approaches for identifying functionally relevant genetic changes.

We develop and maintain a widely used variant effect scoring tool (Combined Annotation Dependent Depletion, CADD), that uses machine learning to integrate more than 80 different gene-based and genome-wide annotations. CADD was the first tool to predict variant deleteriousness for all possible single base-pair alterations genome-wide, while also allowing to score multi-base and insertion/deletion changes.

Further, we collaborate with experimental groups to obtain and analyze experimental measures of non-coding sequence activity, specifically from Massively Parallel Reporter Assays (MPRA). Despite the majority of all mutations affecting non-coding sequences and a growing evidence of substantial phenotypic effects as well as clinical relevance, alterations in these sequences remain less well understood than those in coding regions. The lab uses for example experimental data to infer computational models of regulatory sequence effects (i.e. promoter and enhancer sequences) with the goal of contributing to a better understanding of regulatory sequence function and later integrating regulatory sequence models in the next generation of genome-wide variant scores.</p>

<div align="middle">
  <img src="assets/images/reg_seq.jpg" style="max-width:100%;" alt="Regulatory sequences (promoters and enhancers) are involved in control of gene expression. Mutations (stars) increase or decrease the binding of different proteins (transcription factors) to DNA sequence, thereby changing the amount of RNA which can impact certain phenotypes." />
</div>

<div class="figure-caption">
  Regulatory sequences (promoters and enhancers) are involved in the control of gene expression. Mutations (stars) increase or decrease the binding of different proteins (transcription factors) to DNA sequence, thereby changing the amount of RNA which can impact certain phenotypes.
</div>    

    
 <h3 id="CADD">CADD</h3>

<p> CADD is a tool for scoring the deleteriousness of single nucleotide variants as well as insertion/deletions variants in the human genome.

While many variant annotation and scoring tools are around, most annotations tend to exploit a single information type (e.g. conservation) and/or are restricted in scope (e.g. to missense changes). Thus, a broadly applicable metric that objectively weights and integrates diverse information is needed. Combined Annotation Dependent Depletion (CADD) is a framework that integrates multiple annotations into one metric by contrasting variants that survived natural selection with simulated mutations.

C-scores strongly correlate with allelic diversity, pathogenicity of both coding and non-coding variants, and experimentally measured regulatory effects, and also highly rank causal variants within individual genome sequences. Finally, C-scores of complex trait-associated variants from genome-wide association studies (GWAS) are significantly higher than matched controls and correlate with study sample size, likely reflecting the increased accuracy of larger GWAS.

CADD can quantitatively prioritize functional, deleterious, and disease causal variants across a wide range of functional categories, effect sizes and genetic architectures and can be used prioritize causal variation in both research and clinical settings.

Our latest manuscript describes the updates since the initial publication of CADD and how we envision its use. It was published by Nucleic Acids Research in 2019.</p>
  </div>

</article>

      </div>
    </main><footer class="site-footer h-card">
  <data class="u-url" href="/"></data>

  <div class="wrapper">

<!--    <h2 class="footer-heading">JRG Computational Genome Biology</h2>   -->

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1"><ul class="social-media-list"><li><a href="https://github.com/kircherlab"><svg class="svg-icon"><use xlink:href="/assets/minima-social-icons.svg#github"></use></svg> <span class="username">kircherlab</span></a></li></ul>
</div>

      <div class="footer-col footer-col-3">
        <p></p>
      </div>
    </div>

  </div>

</footer>
</body>

</html>
